lib/main.dart

```diff
@@ -1,15 +1,30 @@
-import 'package:concordium_wallet/state.dart';
+import 'package:concordium_wallet/screens/routes.dart';
+import 'package:concordium_wallet/services/http.dart';
+import 'package:concordium_wallet/services/shared_preferences/service.dart';
+import 'package:concordium_wallet/services/wallet_proxy/service.dart';
+import 'package:concordium_wallet/state/config.dart';
+import 'package:concordium_wallet/state/network.dart';
+import 'package:concordium_wallet/state/services.dart';
+import 'package:concordium_wallet/state/terms_and_conditions.dart';
 import 'package:concordium_wallet/theme.dart';
 import 'package:flutter/material.dart';
-
-import 'package:concordium_wallet/screens/routes.dart';
 import 'package:provider/provider.dart';
 import 'package:shared_preferences/shared_preferences.dart';
 
 void main() {
   runApp(const App());
 }
 
+final config = Config.ofNetworks([
+  const Network(
+    name: NetworkName.testnet,
+    walletProxyConfig: WalletProxyConfig(
+      baseUrl: 'https://wallet-proxy.testnet.concordium.com',
+    ),
+  ),
+]);
+const httpService = HttpService();
```

Comment:
I don't see anything bad in httpclient being here. it might be injected in other classes in the feature too. Ive seen  the same approach in ios projects alot

Reply:
Solved in person

Reply:
Moved it closer to where it's being used. This prevents it from being global.